contents/docs/user-guides/groups.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Groups
+sidebar: Docs
+showTitle: true
```

Comment:
TODO: Remove the other docs, add a redirect here.

Reply:
Old docs

Reply:
I think this isn't resolved.